bridge-token-factory/tests/token_transfer.rs

```diff
@@ -245,101 +327,199 @@ fn test_with_invalid_proof() {
 
     // This deposit event must succeed. Notice that previously a similar deposit
     // was made, but it failed because it had an invalid proof, so this one should succeed.
-    call_borsh!(user, factory_id.deposit(proof)).assert_success();
+    assert!(&rt
+        .block_on(
+            user.call(&worker, factory.id(), "deposit")
+                .deposit(parse_near!("50 N"))
```

Comment:
Where this deposit value is coming from?

Reply:
If don't set any deposit appear following error: ExecutionError("Smart contract panicked: panicked at 'assertion failed: env::attached_deposit() >=\n required_deposit + self.bridge_token_storage_deposit_required', src/lib.rs:329:9")

Changed 50N to 1N (https://github.com/aurora-is-near/rainbow-token-connector/commit/893272e94e78ab51e29d02e870eb2c44af54eae6). We should deposit enough near tokens for our storage. But I don't see any functions in contract which say how much is enough and calculation is not very simple. So I decide that for this test is ok to transfer just some amount of tokens and don't make accurate calculation.

Reply:
fixed: https://github.com/aurora-is-near/rainbow-token-connector/pull/159/commits/87ee9af208bad1f1faafcf4ba8e9aaf48e782e82

If deposit wasn't set the next const were used: https://docs.rs/near-sdk-sim/latest/near_sdk_sim/constant.STORAGE_AMOUNT.html